### DISCUSSION TITLE
log levels

### DISCUSSION THREAD

**Discussion by sezanzeb** [sezanzeb/input-remapper#260] [General]
@jonasBoss how would you call extra verbose logs that might print a lot of messages, and that are entirely based on the user configuration.

I dislike the term "spam"

"event" does not work for discovering devices and parsing macros

"processing" might work, but I'm not sure. Is it processing something when discovering devices? Isn't everything else also falling under the hood of "processing"?

I think "verbose" is more like between debug and info. Maybe all debug logs should be called "verbose", and all spam logs "debug"

"debug2"?

"V", "VV", "VVV" for multiple verbosity levels?

idk

![image](https://user-images.githubusercontent.com/28510156/149657393-b75e0781-3e1b-448b-992c-c444854ed08b.png)

**Comment by sezanzeb** [comment]
I think I don't really care about how to call them as long as I get colorful output. I'll try to play around with giving each file a different color instead. If there is a distinction/grouping based on colors I don't mind calling all of them debug.

**Reply by jonasBoss** [reply]
yeah, that is what i think as well. It does not really matter how to call them. I think having just multiple verbosity levels is quiet common. The color is way more helpful than the name.

**Reply by sezanzeb** [reply]
![image](https://user-images.githubusercontent.com/28510156/149991863-9bc2dc23-4f31-4173-aa5d-d841f0b0c04d.png)

This is so much better. I can't believe I never logged the process name until now.

It generates a color from the filename
